core/debug/src/debug.c

```diff
@@ -60,83 +60,6 @@ void debug_init(void)
     SCnSCB->ACTLR |= 0x2;
 }
 
-static void debug_cx_state(int _indent)
```

Comment:
I'd recommend we fix these to work with the new APIs instead of deleting. The refactoring would go into the refactoring commit, just like the Box ID changes did.